### DISCUSSION TITLE
How to use the "Solid" example from tailwindcss-form's live demo

### DISCUSSION THREAD

**Discussion by rangerscience** [tailwindlabs/tailwindcss#14689] [Help]
The [tailwindcss-forms](https://github.com/tailwindlabs/tailwindcss-forms) [live demo](https://tailwindcss-forms.vercel.app/) shows four examples - unstyled, simple, underline, solid - but, none of the docs show _how_ to use those - I'm not even sure what they're called. Are they themes? Do you specify them somehow when registering the plugin? Are they just examples of combining `tailwindcss-forms` with additional Tailwind classes?

Apologies if this is a "dumb" question - it sure feels like there's something super basic that I'm just missing.

**Comment by wongjn** [comment]
They are examples of combining `@tailwindcss/forms` with additional Tailwind classes.
